frontend/packages/console-app/src/plugin.tsx

```diff
@@ -202,7 +202,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       loader: async () =>
         (
           await import(
-            './components/quick-starts/QuickStartCatalogPage' /* webpackChunkName: "co-quick-start" */
+            './components/quick-starts/QuickStartCatalogPageAsync' /* webpackChunkName: "co-quick-start" */
```

Comment:
Again, I think we can use the same chunk name for all things quick starts?
```suggestion
            './components/quick-starts/QuickStartCatalogPageAsync' /* webpackChunkName: "quick-start" */
```